lib/plan-finder.js

```diff
@@ -259,10 +279,47 @@ class PlanFinder {
         if (!this.isMimeType(outputInstructions.type)) {
             throw new RenditionFormatUnsupportedError(`outputInstructions.type (${outputInstructions.type}) is not propertly formatted`);
         }
+
         const plan = this.BFS(source, outputInstructions);
         return this.formPlan(source, outputInstructions, plan);
     }
 
+    async getImageMetadata(path) {
+        let metadata = {};
+        if (path && (await fs.pathExists(path))) {
+            const command = `exiftool -json "${path}"`;
+
+            try {
+                console.log(`Running command: ${command}`);
+                const stdout = child_process.execSync(command, {
+                    timeout: EXIFTOOL_TIMEOUT_MS
+                });
+                const rawJSON = JSON.parse(stdout)[0];
+                metadata = rawJSON;
+            } catch (error) {
+                console.error(`error getting metadata via exiftool for source file: ${path}`);
```

Comment:
Sometimes, `exiftool` won't throw, but store an error in `stderr`. Maybe you should check that too?
The line of code from L294 would become something like:
`const { stdout, stderr } = child_process.execSync(command, {});`
and you would need to check if stderr is not set/empty or not to be sure no error was missed.